client/stylesheets/_toCat.scss

```diff
@@ -3123,6 +3123,10 @@ On click the images will be displayed at normal size to complete the effect
     font-size: 1.1rem; 
 }
 
+.listingItemDetailItems li a:hover {
```

Comment:
Merge rule `.listingItemDetailItems li a:hover` with rule on line 3114<br>Selector should have depth of applicability no greater than 2, but was 3<br>Selector `listingItemDetailItems` should be written in lowercase with hyphens